src/components/password/password.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useRef } from "react";
+import { TextboxProps } from "../textbox";
+import { StyledPassword, HiddenAriaLive } from "./password.style";
+import guid from "../../__internal__/utils/helpers/guid";
+import useLocale from "../../hooks/__internal__/useLocale";
+
+export type TypeOptions = "text" | "password";
+
+export interface PasswordProps extends TextboxProps {
+  /** HTML type attribute of the input */
+  type?: TypeOptions;
+}
+
+export const Password = ({ id, type, ...rest }: PasswordProps) => {
+  const internalInputId = useRef(id || guid());
+  const l = useLocale();
+  return (
+    <>
+      <StyledPassword
+        data-element="styled-password-container"
+        id={internalInputId.current}
+        autoComplete="off"
+        type={type || "password"}
+        {...rest}
+      />
+      <HiddenAriaLive role="alert" aria-live="polite">
```

Comment:
Not sure if `alert` is the correct role for this as `alert` tends to be for time-sensitive _things_. Not really sure what to use instead though as nothing really fits the needs of a password input. Might be a question for the UX/Accessibility lot. Alert also has `aria-live="assertive"` out the box too so you're overriding this too. We'll see what accessibility thinks 😄

Reply:
Thanks, Dan, have gone for `role="status"`, `aria-live="polite"` for the meantime as I feel the update is not important enough to be an alert in retrospect. But happy to amend/change this down the line.